### ISSUE TITLE
Windscribe VPN breaking Taildrop and Serve

### ISSUE THREAD

**Issue by Kazaflow** [tailscale/tailscale#8472]
### What is the issue?

I use Tailscale to remote into my server, it has a Windscribe VPN running for torrenting. So far both worked okey along each other with the help of excluding Tailscales IP-range and the .exe files with split-tunneling.

I tried using Tailscale serve today and after some trail and error found out that regardless of the split-tunneling the VPN is breaking it.
I want to serve http://127.0.0.1:8080/ and it fails. But then I can serve text and directories with no issues.

Taildrop used to work a few weeks ago, even when the VPN was on. Now it just hangs, disappears and after a while gives an error saying:
"An error was encountered transferring the file:
Put “http://100.76.49.111:1/v0/put/test.txt": dial tcp 100.76.49.111:1: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."

### Steps to reproduce

1. Have Windscribe VPN and Tailscale running.
2. Try to serve a web server or taildrop something from the machine.

### OS

Windows

### OS version

Windows 10 Enterprise LTSC 21H2 19044.3086

### Tailscale version

1.44.0

### Other software

Windscribe Desktop App v2.6.14

### Bug report

BUG-f5ee0e36e5289cbfcab938632c730eec4d874f5218eaa3380e624d8d76ffb7c2-20230628154208Z-2bba0d8e5ccf257a

**Comment by Kazaflow** [comment]
Ah, also it only fails if the serving/taildropping machine has the Windscribe VPN on. If a machine that tries to access served stuff has it on it works fine.

**Comment by DentonGentry** [comment]
Tailscale 1.44.0 was released June 21. Did the issue start after updating Tailscale?
The prior release can be downloaded from https://pkgs.tailscale.com/stable/tailscale-setup-1.42.0.exe if you'd like to check.

The most obvious way taildrop could be broken without breaking all other functionality in Tailscale is the PeerAPI, which is an HTTP listener which the Windows Tailscale Service listens on. Taildrop and exit node DNS service relies on PeerAPI, plus a few more lesser known functions. Most of the rest of the Tailscale functionality still works even if the PeerAPI isn't reachable.

**Comment by Kazaflow** [comment]
Rolling back to 1.42.0 still presents that issue, also have noticed that sometimes one machine doesn't see the other in the Taildrop menu, just before now I didn't see the server from the machine, but now its the other way around. And thats even when the Windscribe VPN isn't active.

Is there anything I can check further to see what could interfere with the PeerAPI?

**Comment by Kazaflow** [comment]
Ok no, something in general seems to be messed up, the server has Jellyfin running and it can be accessed with the server's 192.168.xxx.xxx IP but itself cant connect to localhost. Guess I will have to figure that out.

**Comment by mediaserverr** [comment]
I have the same issue. Has any solution been found?

**Comment by Alkeari** [comment]
This worked for me: "https://tailscale.com/docs/reference/faq/other-vpns#split-tunnels"

Windscribe -> Connection -> Split Tunneling -> On -> Mode: Exclusive -> IPs & Hostnames -> "100.64.0.0/10" & "fd7a:115c:a1e0::/48

TLDR: Set "100.64.0.0/10" & "fd7a:115c:a1e0::/48" for Split Tunneling.
